### ISSUE TITLE
Docs / examples using rest api client

### ISSUE THREAD

**Issue by sasakrsmanovic** [unoplatform/uno#4115]
Docs / examples using rest api client

**Comment by DanielCoder834** [comment]
Is this issue still available?

**Comment by jeromelaban** [comment]
@DanielCoder834 yes, it's available! Did you have to search for docs when using a REST API?

**Comment by DanielCoder834** [comment]
@jeromelaban Truthfully, I am trying to get into open-source development and I think this would be a good first issue to complete.
